### DISCUSSION TITLE
Problem with loading Laravel Nova in production site

### DISCUSSION THREAD

**Discussion by RishabhTayal** [laravel/nova-issues#5751] [Q&A] [answered]
- Laravel Version: 8.83.27
- Nova Version: 4.26.5
- PHP Version: 8.2.7
- Database Driver & Version: MySql

### Description:
I recently migrated to nova 4.26.0. I followed the migration guide as well. https://nova.laravel.com/docs/4.0/upgrade.html. It's now working in local environment. But getting empty page with browser console error in production site.

Here is the error I am getting:

![](https://i.stack.imgur.com/nGp62.png)

**Comment by davidhemphill** [comment]
Ensure the production assets have been published and are being loaded. You should try running `php artisan nova:publish --force` on the production server to make sure they're getting the latest versions.

**Reply by RishabhTayal** [reply]
I have already tried that. Getting same error.

**Reply by davidhemphill** [reply]
Ensure that you removed any layout files from the `resources/views/vendor/nova` folder. Nova's application is not defined, indicating some part of the boot process is not happening. This would typically be due to the layout being modified.

**Reply by RishabhTayal** [reply]
Understood. I tried that but getting same error. There was a layout.blade.php file in `resources/views/vendor/nova` folder. I have deleted that and ran `php artisan nova:publish --force` and `php artisan optimize`. 😞

**Comment by crynobone** [comment]
This is a known issue with using Laravel Nova with Cloudflare Rocket Loader as explained in https://github.com/laravel/nova-issues/discussions/4127

You can tell by looking at `(anonymous function) - rocket-loader.min.js`

**Reply by RishabhTayal** [reply]
Finally some answer that seems like potential solution. I have been trying to solve this for past 3 days. 😭. I will try to add a rule in cloudflare to bypass some urls.

**Reply by RishabhTayal** [reply, marked as answer]
Disabled Rocket Loader in Cloudflare and it started working!!! Thanks a lot @crynobone 

<img width="1042" alt="image" src="https://github.com/laravel/nova-issues/assets/2100166/ee07433d-5a59-4764-a275-5221bfbb2bbf">
